benefits/enrollment/templates/enrollment/index.html

```diff
@@ -45,6 +45,7 @@ <h1 class="pb-lg-8 pb-4">
                 // https://stackoverflow.com/a/42469170
                 // use 'assign' because 'replace' was giving strange Back button behavior
                 window.location.assign(data.redirect);
+                return;
```

Comment:
Why is this needed? I thought it was working in #2089? Maybe add a small comment

Reply:
Ah, I see the commit message now. I think a code comment would help thought!